### PR TITLE
fix(DDIDS-1031): styling changes of callout modal

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Adjusted border radius for callout variant
Adjusted padding for modals without callout
Added modal border and box-shadow properties

### CODE REVIEW

File: libs/web-components/src/components/modal/Modal.svelte [hunk 1]
```diff
@@ -181,9 +182,10 @@
     width: 90%;
     display: flex;
     margin: 1rem;
-    box-shadow: var(--shadow-2);
+    box-shadow: 6px 6px 6px rgba(0, 0, 0, 0.16);
     border-radius: 4px;
     max-height: 90%;
+    border: 1px var(--color-gray-600);
```
Comment (chrisolsen) [inline libs/web-components/src/components/modal/Modal.svelte:188]: you are missing the pattern definition
```
border: 1px solid var(--color-gray-600)
```

Comment (Radha-Alberta) [reply libs/web-components/src/components/modal/Modal.svelte]: Changed.

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**